_includes/global_fields.md

```diff
@@ -4,6 +4,7 @@ A set of common fields that appear on all resources is included first, for refer
 
 | Name          | Type      | Description
 |-----------    |-----------|-----------|--------------
+|id 			| string    | A string representing a local identifier for the resource on the local system.  It may be a string, integer, guid, etc.  This value MUST be suitable for use in a helper function such as add_tags, add_lists to identify the corresponding resource
```

Comment:
Do we want to constraint this id so that it's the same as the one in the resource path?  I can see pros/cons to that constraint, biggest pro being much less work to do in updating all the old docs. :)